src/main/java/com/example/booksstore/service/impl/UserServiceImpl.java

```diff
@@ -31,6 +33,7 @@ public UserResponseDto register(UserRegistrationRequestDto user) {
                         + Role.RoleName.USER
                         + " is not exists."));
         User model = userMapper.toModel(user);
+        model.setPassword(passwordEncoder.encode(model.getPassword()));
```

Comment:
You should set default role on registation

Reply:
Merged